src/flow/src/metrics.rs

```diff
@@ -28,6 +28,32 @@ lazy_static! {
         &["table_id"]
     )
     .unwrap();
+    pub static ref METRIC_FLOW_BATCHING_ENGINE_QUERY_TIME: HistogramVec = register_histogram_vec!(
+        "greptime_flow_batching_engine_query_time",
+        "flow batching engine query time",
+        &["flow_id"],
+        vec![
+            0.0,
+            1.,
+            3.,
+            5.,
+            10.,
+            20.,
+            30.,
+            60.,
+            2. * 60.,
+            5. * 60.,
+            10. * 60.
+        ]
+    )
+    .unwrap();
```

Comment:
we usually use a fixed step, like 2, 3 or 10. E.g.:
```
vec![1, 2, 4, 8]
```

And btw what's the unit of this metric?